commands-yml/commands/device/app/get-app-strings.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Get App Strings
+short_description: Get app strings
+example_usage:
+  java:
+    |
+      Map<String> appStrings = driver.getAppStringMap("en", "/path/to/file");
+  python:
+    |
+      appStrings = self.driver.app_strings("en", "/path/to/file")
+  javascript_wd:
+    |
+      let appStrings = await driver.getAppStrings();
+  javascript_wdio:
+    |
+      let appStrings = driver.getAppStrings("en");
+  ruby:
+    |
+      @driver.app_strings("en")
+  php:
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "http://appium.github.io/java-client/io/appium/java_client/HasAppStrings.html#getAppStringMap--"
+  python: "https://github.com/appium/python-client/blob/master/appium/webdriver/webdriver.py#L357"
+  javascript_wdio: "http://webdriver.io/api/mobile/getAppStrings.html"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L2916"
+  ruby: "http://www.rubydoc.info/github/appium/ruby_lib/Appium/Core/Device:app_strings"
+  php: "https://github.com/appium/php-client/" # TODO PHP documentation link
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO Dotnet documentation link
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: false
+  windows:
+    windows: false
+client_support:
+  java: true
+  python: true
+  ruby: true
+  php: true
+  csharp: true
+  javascript_wd: true
+  javascript_wdio: true
+  
+
+# Information about the HTTP endpoints
+endpoint:
+  url: /session/:session_id/appium/app/strings
+  method: 'POST'
+  url_parameters:
+    - name: session_id
+      description: ID of the session to route the command to
+  json_parameters:
+    - name: language
```

Comment:
Do we have no way of indicating required vs. optional payload parameters?

Reply:
Not yet. I can add that in a future PR

Reply:
I just added 'optional' in parantheses